src/app/case/case-feature/case-feature.config.ts

```diff
@@ -0,0 +1,107 @@
+import { Injectable } from '@angular/core'
+import { Http } from '@angular/http'
+import { AbstractAppConfig, CaseEditorConfig, CaseEventData } from '@hmcts/ccd-case-ui-toolkit'
+// import { environment as mainConfig } from '../environments/environment'
+//    'case_data_url': 'http://localhost:3001/api/data',
+@Injectable()
+export class AppConfig extends AbstractAppConfig {
+    protected config: CaseEditorConfig = {
+        'api_url': '/aggregated',
```

Comment:
suggest moving this to environment file.

Reply:
yes good idea - moved it